api/python/notebooks/experimental/mean_variance.ipynb

```diff
@@ -0,0 +1,236 @@
+{
```

Comment:
notebook comments:
* blank cell at the end of the file should be deleted
* suggest adding example cell at end that joins with the axis dataframe (eg. obs) and displays the mean/var in that context. This will be the typical usage so we should highlight it.